src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,189 @@
 
   .. container:: infospec
 
-    **TODO**
+    .. container:: heading3
+
+      Understanding component references
+
+    As with references to units and variables, the term "component reference" refers to places where you need to specify a component by name.
+    This is done through the :code:`component_ref` attribute found in :code:`encapsulation` and :code:`import component` elements, as well as the :code:`component_1` and :code:`component_2` attributes of :code:`connection` elements.
+
+    The complicated part occurs during imports, and involves understanding the *scope* of a component's reference.
+    This is really just the appropriate name by which to call it depending on who is doing the calling.
+
+    Consider the following example.  
+    Here, two families live next door to one another, and within each family unit use the terms "husband" and "wife" to refer to one another.
```

Comment:
What do you mean by "... each family unit use..."?

Reply:
I guess I could remove the "unit"?  Would that help it make more sense to you?  It means that within the family of LeadbetterFamily or GoodFamily, the variables are referred to as "husband" and "wife".

Reply:
Yes, it would make sense to me if "unit" was removed (and an "s" was added to "use").

Reply:
Done in bd6d599

Reply:
(actually, slightly different tweaking before I read your message, but I think it's clearer now).